### PR TITLE
Fix Quote

### PR DESCRIPTION
**Pull request by IgorWounds**
Adds support for multiple symbols.

### CODE REVIEW

**Review by jmaslek — APPROVED** [review]

File: openbb_platform/providers/intrinio/openbb_intrinio/models/equity_quote.py [hunk 1]
```diff
@@ -118,18 +119,26 @@ def extract_data(
...
+
+        return results
 
     @staticmethod
     def transform_data(
         query: IntrinioEquityQuoteQueryParams, data: dict, **kwargs: Any
     ) -> IntrinioEquityQuoteData:
         """Return the transformed data."""
-        return IntrinioEquityQuoteData.model_validate(data)
+        return [IntrinioEquityQuoteData(**d) for d in data]
```
Comment (the-praxs) [inline openbb_platform/providers/intrinio/openbb_intrinio/models/equity_quote.py:144]: This should be `[IntrinioEquityQuoteData.model_validate(d) for d in data]`

**Review by the-praxs — CHANGES_REQUESTED** [review]

File: openbb_platform/providers/intrinio/openbb_intrinio/models/equity_quote.py [hunk 2]
```diff
@@ -118,18 +119,26 @@ def extract_data(
...
-        return get_data_one(url, **kwargs)
+        with ThreadPoolExecutor() as executor:
+            executor.map(get_data, [s.strip() for s in query.symbol.split(",")])
+
+        return results
 
     @staticmethod
     def transform_data(
         query: IntrinioEquityQuoteQueryParams, data: dict, **kwargs: Any
     ) -> IntrinioEquityQuoteData:
```
Comment (the-praxs) [inline openbb_platform/providers/intrinio/openbb_intrinio/models/equity_quote.py:142]: This should be `List[IntrinioEquityQuoteData]`

**Review by the-praxs — CHANGES_REQUESTED** [review]

File: openbb_platform/providers/intrinio/openbb_intrinio/models/equity_quote.py (hunk 1 above)
Comment (the-praxs) [reply openbb_platform/providers/intrinio/openbb_intrinio/models/equity_quote.py]: Done!

File: openbb_platform/providers/intrinio/openbb_intrinio/models/equity_quote.py (hunk 2 above)
Comment (the-praxs) [reply openbb_platform/providers/intrinio/openbb_intrinio/models/equity_quote.py]: Done!

**Review by the-praxs — APPROVED** [review]
LGTM 🚀

**Pull request merged**